### ISSUE TITLE
Does this support images?

### ISSUE THREAD

**Issue by tal32123** [Mariuxtheone/Teleport#33]
Does this support sending and displaying images as well, or just strings? I know there is an issue with Android Wear and sending/receiving large files > 100kb.

**Comment by raffaeu** [comment]
Yes you can "stream" or just serialize the image but be aware of the limitations of Android Wear itself
